Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -624,6 +624,35 @@ lemma nonsingular_add {x₁ x₂ y₁ y₂ : F} (h₁ : W.Nonsingular x₁ y₁)
   nonsingular_neg <| nonsingular_negAdd h₁ h₂ hxy
 #align weierstrass_curve.nonsingular_add WeierstrassCurve.Affine.nonsingular_add
 
+variable {x₁ x₂ : F} (y₁ y₂ : F) (hx : x₁ ≠ x₂)
+
+/-- The formula `x(P₁ + P₂) = x(P₁ - P₂) - ψ(P₁)ψ(P₂) / (x(P₂) - x(P₁))²`,
+where `ψ(x,y) = 2y + a₁x + a₃`. -/
```

Comment:
I find this more readable (in VS Code), so I switched from LaTeX to Unicode. I can remove the backticks if you prefer. (I know that backticks can are intended for code blocks in markdown but sometimes I just want to use it for the highlighted monospace style, not necessarily code.)

Reply:
I would argue that ultimately we want equations to be rendered best in mathlib4_docs. Personally I don't look at the code for equations, otherwise my module docstrings in e.g. `DivisionPolynomial/Basic` would be a huge mess.

Reply:
LaTeX won't render when you hover your mouse over the name of the lemma ...
This will still be pretty readable in the mathlib4_docs IMO, so I think it's a good compromise ...